src/platform/forms/containers/review/SubmitController.jsx

```diff
@@ -0,0 +1,232 @@
+// libs
+import React, { useEffect, useState } from 'react';
```

Comment:
summarize changes made to this file

Reply:
This is an attempt to reduce the previous `SubmitController` and `SubmitButtons` components. We are moving all of the logic of what is shown into the `SubmitController` and allowing the components it renders to be their own, re-usable versions.